broker/verboice.config

```diff
@@ -32,6 +32,7 @@
     {guisso_url, ""},
 
     {twilio_callback_url, "http://localhost:8080/"},
+    {twilio_base_url, "https://api.twilio.com/2010-04-01"},
```

Comment:
Will this need to be set up in all instances or these config are used as a fallback when no env var are defined?

Reply:
> these configs are used as a fallback when no env var is defined

Sincerely I can't guaranty this, but I think we can suppose this is the expected behavior since I didn't have to set an env var in my local instance to make it work.

We'll try to test this by deploying a feature branch in staging.